test/integration/certificates/trigger_controller_test.go

```diff
@@ -146,41 +194,30 @@ func TestTriggerController_RenewNearExpiry(t *testing.T) {
 	stopController := framework.StartInformersAndController(t, factory, cmFactory, c)
 	defer stopController()
 
-	// Create a Certificate resource and wait for it to have the 'Issuing' condition.
-	cert, err := cmCl.CertmanagerV1().Certificates(namespace).Create(ctx, &cmapi.Certificate{
-		ObjectMeta: metav1.ObjectMeta{Name: "testcrt", Namespace: namespace},
-		Spec: cmapi.CertificateSpec{
-			SecretName: "example",
-			CommonName: "example.com",
-			IssuerRef:  cmmeta.ObjectReference{Name: "testissuer"}, // doesn't need to exist
-		},
-	}, metav1.CreateOptions{})
+	// Create a Certificate
+	cert, err = cmCl.CertmanagerV1().Certificates(namespace).Create(ctx, cert, metav1.CreateOptions{})
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	// Ensure that the Certificate does *not* get the Triggered status condition
-	// if the status.renewalTime is not set.
+	// 1. Test that Certificate's Issuing condition is not set to True when the x509 cert is not approaching expiry
```

Comment:
I suggest that we keep our comments widths short enough so that it feets the editor without having to soft wrap text 😅 I use [rewrap](https://github.com/stkb/Rewrap) to keep my comments under 80 chars, but 100 is fine too

```suggestion
	// 1. Test that the Certificate's Issuing condition is not set to True when the
	// X.509 cert is not approaching expiry.
```

Reply:
Cool! I will check out the rewrap tool as well! Agree that it's more readable that way.